test/src/main/java/io/strimzi/test/k8s/cmdClient/Kubectl.java

```diff
@@ -32,6 +32,11 @@ public String defaultNamespace() {
         return "default";
     }
 
+    @Override
+    public String defaultOlmNamespace() {
+        return "operators";
```

Comment:
Isnt the default namespace openshift-operators ?

Reply:
TBH I am not sure. I just read some blog about operatorhub on Kubernetes but I didn't try it. Now I am looking at Strimzi manifests available at https://operatorhub.io/operator/strimzi-kafka-operator and according to [this link](https://operatorhub.io/install/strimzi-kafka-operator.yaml) it should be `operators` for Kubernetes.